developer_manual/examples/cron/SomeTask.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace OCA\MyApp\Cron;
+
+use \OCA\MyApp\AppInfo\Application;
+
+class SomeTask
+{
+    public static function run()
+    {
+        (new Application())
+            ->getContainer()
+            ->query('SomeService')
+            ->run();
+    }
```

Comment:
the brackets doesn't follow our coding style, the opening bracket must be on the same line as the function signature. See https://doc.owncloud.org/server/9.1/developer_manual/general/codingguidelines.html#objects-functions-arrays-variables